vocabs/oada/isoblue/index.js

```diff
@@ -0,0 +1,72 @@
+var libvocab = require('../../../lib/vocab')('oada');
+// Only importing the essentials, more complex definitions may need more imports
+// similar to the main oada index.js this is based off of
+const {register,  override, enumSchema} = libvocab;
+
+// Custom location schema
+register('lat', override('latitude', {
+    description: 'Short form of latitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+  
+register('lng', override('longitude', {
+    description: 'Short form of longitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+
+
+// Custom CAN schema
+register('is_error_frame',  {
+    description: 'Error frame flag for can frame',
+    type: Boolean,

```

Comment:
Removed `generic-data-value` overrides for can frame details as they are not really data values, but components of a CAN Frame with specific data types